gobblin-core/src/main/java/gobblin/writer/AvroHdfsDataWriter.java

```diff
@@ -167,6 +168,24 @@ public void commit()
     if (!this.fs.rename(this.stagingFile, this.outputFile)) {
       throw new IOException("Failed to commit data from " + this.stagingFile + " to " + this.outputFile);
     }
+
```

Comment:
I suggest abstracting the following code into a utility method into `gobblin.util.HadoopUtils`, say, `public static void keepSourceFileAttributes(FileSystem fs, Path file, State properties)`.